Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -10096,6 +10129,17 @@
         }
       }
     },
+    "reportSendToNosConfirmation" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Nos will analyze the note for %@ content and if we agree will label the note anonymously. If the label doesn't match the NIP-56 for the category we may choose a different label or no label at all."
```

Comment:
I find this confusing on two points:
1. If Nos labels the note anonymously then how will people know to trust the events? I think what we are trying to say is that **you**, the user, will remain anonymous, but Nos will publish it from our Reportinator account.
2. "If the label doesn't match the NIP-56 for the category" - this doesn't make sense to me, and I don't think users will know what the difference between "label" and "category" mean to or what NIP-56 is.  

Here's a take at solving both issues:
"The Nos moderation team will analyze the note for %@ content and may publish a report from our own account, concealing your identity."

Reply:
Going to loop in @setch-l for proposed copy changes.

Reply:
Added 
<img width="426" alt="Screenshot 2024-04-29 at 3 49 34 PM" src="https://github.com/planetary-social/nos/assets/18184/c024c6d8-9af1-4fc3-b83e-955b399c23d2">